src/VisualStudio/DevKit/Impl/Microsoft.VisualStudio.LanguageServices.DevKit.csproj

```diff
@@ -5,7 +5,6 @@
     <OutputType>Library</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
-    <IsPackable>true</IsPackable>
```

Comment:
We were creating a NuGet package for this project, but we don't actually need it. We make an NPM package, but we can turn this off and it still works. This had to be fixed or otherwise when building this package, it no longer knows which of the LanguageServer packages to reference.